.claude/skills/new-guided-blog-post/SKILL.md

```
@@ -41,6 +41,26 @@ Always apply Peter's writing voice from `.claude/context/writing-style.md`:
+
+1. `git branch --show-current` — note current branch
+2. `git status --short` — must be clean; if not, stop and warn the user
+3. If already on a `feat/blog/*` branch, ask whether to extend it or cut a new one
+
+Create the branch:
+
+```bash
+git checkout master
+git pull origin master
+git checkout -b feat/blog/<short-slug>
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)